lnrpc/walletrpc/psbt.go

```diff
@@ -42,29 +43,41 @@ func verifyInputsUnspent(inputs []*wire.TxIn, utxos []*lnwallet.Utxo) error {
 
 // lockInputs requests a lock lease for all inputs specified in a PSBT packet
 // by using the internal, static lock ID of lnd's wallet.
-func lockInputs(w lnwallet.WalletController,
-	outpoints []wire.OutPoint) ([]*base.ListLeasedOutputResult, error) {
+func lockInputs(w lnwallet.WalletController, outpoints []wire.OutPoint,
+	customLockID *wtxmgr.LockID, lockExpirationSeconds uint64) (
```

Comment:
Could make the expiration a `time.Duration` here, then we don't need the `Seconds` name suffix.

Reply:
Done.